ukat/data/fetch.py

```diff
@@ -304,6 +304,26 @@
                                        ['050c7ef07574d893b3511796050748fe'],
                                        doc='Downloading Philips tSNR data')
 
+fetch_pc_left_philips = _make_fetcher('fetch_pc_left_philips',
```

Comment:
Could this be moved so the fetch functions are in alphabetical order i.e. move this block to between `fetch_mtr_philips` and `fetch_t1_philips_1`.